activiti-cloud-notifications-graphql/values.yaml

```diff
@@ -91,10 +91,13 @@ terminationGracePeriodSeconds: 20
 ingress:
   ## Set to true to enable ingress record generation
   enabled: true
-  path: # configure base path for ingresses, i.e. {{ .Release.Name }}
+  path: # configure base path for ingresses, i.e. /{{ .Release.Name }}
   tls: # true | false 
   tlsSecret: # secret name of the gateway host, tls-activiti-cloud-gateway
   hostName: # override .Values.global.gateway.host
+  actuator: 
```

Comment:
@igdianov based on the description of the PR I was expecting to only see this change in the template, that is why I am checking the ingress path changes as well.

Reply:
@salaboy There are no path changes for ingress. I have corrected a typo in the comments.

Reply:
@salaboy  I thought it would make sense to make Ingress configurable, i.e. disable | enable `/graphiql` or `/actuator` if someone wants. I think it is a good practice to make all values explicit, but simplify configuration using templates with feature toggles. I like to have opinionated configuration but without hard coding the templates.